CMakeLists.txt

```diff
@@ -0,0 +1,119 @@
+cmake_minimum_required(VERSION 2.8)
```

Comment:
`project` command should be right after `cmake_minimum_required`:
* http://cgold.readthedocs.io/en/latest/tutorials/project.html#when-not-declared